exercise2.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+from matplotlib.widgets import *
+
+
+class Plotter:
+    fig, ax = plt.subplots()
+    plt.subplots_adjust(bottom=0.2)
+    
+    experiment_name = ''
+
+    grid = False
+
+    def the_function(self, mul):
+        return 3 * np.pi * np.e ** (-5 * np.sin(2 * np.pi * mul * self.t))
+
+    def toggle_grid(self, event):
+        self.grid = not self.grid
+        print('grid', self.grid)
```

Comment:
```suggestion
        print('grid: {}'.format(self.grid))
```

`print(a, b)` works good for quick code, but I wouldn't rely on it for a finished script. And if you are using Python3 (which from the README you are), I would even suggest you to use f-strings